pkg/db/sqlite.go

```diff
@@ -117,18 +117,18 @@ func (scf *sqliteConnectionFactory) Init(_ bool) error {
 		//read DDL (test-table structure)
 		ddl, err := ioutil.ReadFile(scf.schemaFile)
 		if err != nil {
-			return err
+			return errors.Wrapf(err, "error reading file DDL schema file '%s'", scf.schemaFile)
```

Comment:
Please check my comments regarding this `errors` external package.

Reply:
I reopened this comment as bookmark, to address some potential tech dept, and how we deal with it. Let's discuss it also in our meeting.